srcpkgs/nyxt/template

```diff
@@ -6,8 +6,8 @@ create_wrksrc=yes
 build_style=gnu-makefile
 make_build_target=all
 hostmakedepends="sbcl git"
-makedepends="webkit2gtk-devel libfixposix-devel libgirepository-devel"
-depends="dbus xclip enchant2 webkit2gtk-devel libfixposix-devel libgirepository-devel"
+makedepends="webkit2gtk libfixposix-devel libgirepository-devel"
+depends="dbus xclip enchant2 webkit2gtk libfixposix libgirepository"
```

Comment:
We may want to add this line to template:
```
shlib_requires="libwebkit2gtk-4.0.so.37 libfixposix.so.3 libgirepository-1.0.so.1"
```

Reply:
It's been added. I did think the depends entries would handle it?

Reply:
It will prevent accidental update of webkit2gtk fixposix gi-repository that change SONAME.